.github/workflows/build-docker.yaml

```
@@ -37,10 +43,19 @@ jobs:
 
+      - name: Get docker tag
+        id: vars
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            echo "TAG=pr-${{ github.event.pull_request.number }}" | tee -a $GITHUB_OUTPUT
+          else
+            echo "TAG=latest" | tee -a $GITHUB_OUTPUT
+          fi
+
       - name: Build and push docker image
         uses: docker/build-push-action@v4
```

actionlint (1.7.7)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)